File: leetcode/2348.py
class Solution:
    def zeroFilledSubarray(self, nums: List[int]) -> int:
        ans = 0
        nums.append(1)
        zeros = 0
        for num in nums:
            if num == 0:
                zeros += 1
            else:
                ans += zeros * (zeros+1) // 2
                zeros = 0
        return ans

File: leetcode/2348.cpp
class Solution
{
public:
    long long zeroFilledSubarray(vector<int> &nums)
    {
        long long ans = 0, zeros = 0;
        for (auto x : nums)
        {
            if (x)
            {
                zeros = 0;
            }
            else
            {
                ans += zeros + 1;
                ++zeros;
            }
        }
        return ans;
    }
};